src/pudl/io_managers.py

```diff
@@ -322,13 +322,21 @@ def load_input(self, context: InputContext) -> pd.DataFrame:
 class PudlParquetIOManager(IOManager):
     """IOManager that writes pudl tables to pyarrow parquet files."""
 
+    def _get_table_resource(self, table_name: str) -> Resource:
```

Comment:
This handles metadata for PUDL models tables slightly differently from all other tables.